docs/resources/app.md

```diff
@@ -11,28 +11,28 @@ subcategory: "Apps"
 
 ```hcl
 resource "databricks_app" "this" {
-  name             = "my-custom-app"
-  description      = "My app"
+  name        = "my-custom-app"
+  description = "My app"
   resources = [{
```

Comment:
Interesting. This first resource is less indented than the others, maybe due to some strange logic with the `[` and `{` on the same line. Can you put a newline there and run it once more?

Reply:
adding a new line doesn't change the behavior. Also, I tested with putting the code into a terraform file and running `terraform fmt` on it - it's formatted it the same way as with `fmt-docs`